README.md

```diff
@@ -0,0 +1,99 @@
+# InfluxDB Module
+
+|                |                                |
+| -------------- | ------------------------------ |
+| Name           | InfluxDB Module                |
+| Version        | v1.0.0                         |
+| Dockerhub Link | [weevenetwork/influxdb-http]() |
+| Authors        | Mesud Pasic                    |
+
+- [MQTT Ingress](#mcclimate-decoder)
+  - [Description](#description)
+  - [Features](#features)
+  - [Environment Variables](#environment-variables)
+    - [Module Specific](#module-specific)
+    - [Set by the weeve Agent on the edge-node](#set-by-the-weeve-agent-on-the-edge-node)
+  - [Dependencies](#dependencies)
+
+## Description
+
+HTTP module for accessing influxDB data
```

Comment:
Please describe in detail what the module does, so that external developer/engineer can understand without reading code.

Reply:
done.